client/app/components/viz/VariantInspectCard.vue

```diff
@@ -967,6 +1052,27 @@ export default {
       }
       return popAF;
     },
+    togglePedigreePopup() {
+      if (this.showPedigreePopup) {
+        this.showPedigreePopup = false;
+      } else {
+        this.showPedigreePopup = true;
+      }
+    },
+    setNumOfSiblings() {
```

Comment:
I try to avoid setting additional variables in a Vue component unless it is absolutely necessary. The fewer variables, the easier it is to understand the code and the less likely the component is to break with additional changes. How about a condition like this?
`v-if="Object.keys(pedigreeData).length > 5"`

Reply:
This change is completed removed the siblings function and data attribute and replaced with this logic. Appears to render as expected.